src/vmshepherd/presets/abstract.py

```diff
@@ -16,11 +29,41 @@ def __init__(self, runtime, defaults):
 
     async def _get_preset_spec(self, preset_name: str):
         """ Returns configuration of specific preset
+
+        :arg string preset_name: Name of the preset
+
+        Returns dict - preset configuration
+
+        Example configuration:
+
+        ::
+
+            name: 'C_DEV-app-dev'
+            count: 1
+            flavor: m1.small
+            image: ubuntu-xenial
+            manage_interval: 0    # interval in seconds
+            manage_expire: 120    # max time of manage (after this time another manage can be scheduled) - expiration in seconds
+            network:
+              security_groups:
+                - PaaS PrivateCloud
+                - paas
+              availability_zone: nova
+              subnet:
+                - pl-krk-2-something
+            iaas:
+              project_name: PROJECT_NAME
+              project_domain_name: PROJECT_DOMAIN
+            userdata: ''
+            meta_tags: {}
+
         """
         raise NotImplementedError
 
     async def _list(self):
-        """ Returns list of configured presets
+        """ Returns list names of presets to manage
+
+        Returns list of string. Name will be used to fetch configuration using `_get_preset_spec`.
```

Comment:
Hm we can use this method not only for gathering configuration. I am right? 
So maybe we can write this docstring like that:
Method collect information about presets managed by application
Returns list of strings. Every item of list is a preset name

Returns list of strings : Every element of this list is a preset name

Reply:
This method should only returns list of presets to be manged.

Reply:
sounds better :)